scripts/gha/test_simulator.py

```diff
@@ -493,14 +493,21 @@ def _setup_android(platform_version, build_tool_version, sdk_id):
     os.path.join(android_home, "platform-tools"), 
     os.path.join(android_home, "build-tools", build_tool_version)]
   os.environ["PATH"] += os.pathsep + os.pathsep.join(pathlist)
-  
+
   args = ["sdkmanager", 
     "emulator", "platform-tools", 
     "platforms;%s" % platform_version, 
     "build-tools;%s" % build_tool_version]
   logging.info("Install packages: %s", " ".join(args))
   subprocess.run(args=args, check=True)
 
+  args = ["sdkmanager", "--licenses"]
+  logging.info("Accept all licenses: %s", " ".join(args))
+  p_yes = subprocess.Popen(["echo", "yes"], stdout=subprocess.PIPE)
```

Comment:
Wow is this really the right way to do this?

Shouldn't it be more like ... create a file handle, pass it into the below subprocess.Popen, and then write "yes" to it in Python?

Reply:
I'm no expert. This is a headache for me. It definitely works here: https://github.com/firebase/firebase-cpp-sdk/blob/9fce79ac551c04c352c8d511ef340c13928d1697/scripts/gha/test_simulator.py#L531

What's your solution? Could you provide an example?

Reply:
I'm not really an expert either - let's save this for a future cleanup. :)